pynamodb/connection/base.py

```diff
@@ -529,8 +531,12 @@ def create_table(self,
             PROVISIONED_THROUGHPUT: {
                 READ_CAPACITY_UNITS: read_capacity_units,
                 WRITE_CAPACITY_UNITS: write_capacity_units
-            }
+            },
+            BILLING_MODE: billing_mode
         }
+        if operation_kwargs.get(BILLING_MODE) == PAY_PER_REQUEST_BILLING_MODE:
+            operation_kwargs.pop(PROVISIONED_THROUGHPUT, {})
```

Comment:
nit - why not just `del operation_kwargs[PROVISIONED_THROUGHPUT]`?

Reply:
^

Reply:
pop is easier though and we can handle the case where if the `PROVISIONED_THROUGHPUT` key is not present easing any future changes